src/ch00/cairo_cheatsheet/list.md

```diff
@@ -0,0 +1,65 @@
+# List
+
+An ordered sequence of values that can be used in Starknet storage:
```

Comment:
Mention Alexandria first, and add a link to https://github.com/keep-starknet-strange/alexandria/tree/main/src/storage